add caveat about messaging btw deprecation warnings and dbt-autofix (#8270)
fishtown-analytics/docs.getdbt.com: website/docs/guides/fusion-upgrade.md

   - **No warnings found**: Skip to Step 4 to continue upgrading.
   - **Warnings found**: Continue to Step 3 to resolve them.

> :::info Conflicting compatibility messages

Author: ```suggestion
:::info Inconsistent Fusion warnings and `dbt-autofix` logs
```

> You may see deprecation warnings about packages not being compatible with Fusion, while **dbt autofix** indicates they are compatible. **dbt autofix is the source of truth**—it has additional context that deprecation warnings don't yet have. This conflict is temporary and will be resolved within a couple of weeks.

Author: ```suggestion
You may see <Constant name="fusion" /> deprecation warnings about packages not being compatible with <Constant name="fusion" />, while **dbt autofix** indicates they are compatible. Use **dbt autofix as the source of truth** because it has additional context that <Constant name="fusion" /> warnings don't yet have. This conflict is temporary and will be resolved as soon as we implement and roll-out `dbt-autofix`'s enhanced compatibility detection to <Constant name="fusion" /> warnings. 
```

> You may see <Constant name="fusion" /> deprecation warnings about packages not being compatible with <Constant name="fusion" />, while `dbt autofix` indicates they are compatible. Use `dbt autofix` as the source of truth because it has additional context that <Constant name="fusion" /> warnings don't yet have. This conflict is temporary and will be resolved as soon as we implement and roll-out `dbt-autofix`'s enhanced compatibility detection to <Constant name="fusion" /> warnings. 

Reviewer: ```suggestion
You may see <Constant name="fusion" /> deprecation warnings about packages not being compatible with <Constant name="fusion" />, while `dbt autofix` indicates they are compatible. Use `dbt autofix` as the source of truth because it has additional context that <Constant name="fusion" /> warnings don't have yet. This conflict is temporary and will be resolved as soon as we implement and roll out `dbt-autofix`'s enhanced compatibility detection to <Constant name="fusion" /> warnings. 
```

> You may see <Constant name="fusion" /> deprecation warnings about packages not being compatible with <Constant name="fusion" />, while `dbt autofix` indicates they are compatible. Use `dbt autofix` as the source of truth** because it has additional context that <Constant name="fusion" /> warnings don't yet have. This conflict is temporary and will be resolved as soon as we implement and roll-out `dbt-autofix`'s enhanced compatibility detection to <Constant name="fusion" /> warnings. 

Author: ```suggestion
You may see <Constant name="fusion" /> deprecation warnings about packages not being compatible with <Constant name="fusion" />, while `dbt autofix` indicates they are compatible. Use `dbt autofix` as the source of truth because it has additional context that <Constant name="fusion" /> warnings don't yet have. This conflict is temporary and will be resolved as soon as we implement and roll-out `dbt-autofix`'s enhanced compatibility detection to <Constant name="fusion" /> warnings. 
```

> You may see <Constant name="fusion" /> deprecation warnings about packages not being compatible with <Constant name="fusion" />, while **dbt autofix** indicates they are compatible. Use **dbt autofix as the source of truth** because it has additional context that <Constant name="fusion" /> warnings don't yet have. This conflict is temporary and will be resolved as soon as we implement and roll-out `dbt-autofix`'s enhanced compatibility detection to <Constant name="fusion" /> warnings. 

Author: ```suggestion
You may see <Constant name="fusion" /> deprecation warnings about packages not being compatible with <Constant name="fusion" />, while `dbt autofix` indicates they are compatible. Use `dbt autofix` as the source of truth** because it has additional context that <Constant name="fusion" /> warnings don't yet have. This conflict is temporary and will be resolved as soon as we implement and roll-out `dbt-autofix`'s enhanced compatibility detection to <Constant name="fusion" /> warnings. 
```
